packages/rest-crud/README.md

```diff
@@ -29,18 +28,79 @@ npm install --save @loopback/rest-crud
    >(Product, {basePath: '/products'});
    ```
 
-4. Set up dependency injection for the ProductController.
+2. Set up dependency injection for the ProductController.
 
    ```ts
    inject('repositories.ProductRepository')(ProductController, undefined, 0);
    ```
 
-5. Register the controller with your application.
+3. Register the controller with your application.
 
    ```ts
    app.controller(ProductController);
    ```
 
+## Creating a CRUD repository
+
+Use the `defineCrudRepositoryClass` method to create named repositories (based
+on the Model) for your app.
+
+Usage example:
+
+```ts
+const db = new juggler.DataSource({connector: 'memory'});
+const ProductRepository = defineCrudRepositoryClass(Product);
+const repo = new ProductRepository(db);
+```
+
+## Integrated example
+
+Here is an example of an app which uses `defineCrudRepositoryClass` and
+`defineCrudRestController` to fulfill its repository and controller
+requirements. You don't need to run `lb4 model` and `lb4 repository` to generate
+the model and respository classes.
+
+```ts
+class Product extends Entity {
+  static definition = new ModelDefinition('Product')
+    .addProperty('id', {type: 'number', id: true})
+    .addProperty('name', {type: 'string'});
```

Comment:
This is duplicating the information provided via `@property` decorators. Please remove the static property `definition` and decorate the class with `@model` decorator instead. That will take care of populating `Product.definition` from metadata provided via `@property` decorators.